test/ThreeFourteen.Finnhub.Client.Tests/Stocks.cs

```diff
@@ -205,7 +205,7 @@ public async Task Candles_WithRange()
 
             var client = new FinnhubClient(httpClientTester.Client, "APIKey");
 
-            var candles = await client.Stock.GetCandles("AAPL", Resolution.FiveMinutes, DateTime.Parse("2019/12/1"), DateTime.Parse("2019/12/2"));
+            var candles = await client.Stock.GetCandles("AAPL", Resolution.FiveMinutes, new DateTimeOffset(2019, 12, 1, 0, 0, 0, TimeSpan.Zero).UtcDateTime, new DateTimeOffset(2019, 12, 2, 0, 0, 0, TimeSpan.Zero).UtcDateTime);
```

Comment:
Might be better with:
```c#
            var candles = await client.Stock.GetCandles("AAPL", Resolution.FiveMinutes, 
                DateTime.Parse("2019/12/1 00:00:00Z"), 
                DateTime.Parse("2019/12/2 00:00:00Z"));

```
But if you're happy it works, I'm happy to merge.
Also it's a pain in the backside for me to test so I'm happy to go with this.

Reply:
Yes, probably the outcome would be equivalent and it's a matter of taste - yours is shorter obviously. I just went this way because I was sure it works this way and with anything except `DateTimeOffset` I would have needed to think more about `Kind` etc.